api.planx.uk/webhooks/sanitiseApplicationData/index.ts

```diff
@@ -0,0 +1,46 @@
+import SlackNotify from 'slack-notify';
+import { Request, Response, NextFunction } from "express";
+
+import { getOperations, operationHandler } from "./operations";
+import { OperationResult } from "./types";
+
+/**
+ * Called by Hasura cron job `sanitise_application_data` on a nightly basis
+ * See hasura.planx.uk/metadata/cron_triggers.yaml
+ */
+export const sanitiseApplicationData = async (
+  _req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<Response | void> => {
+  const operations = getOperations();
+  let results: OperationResult[] = [];
+  try {
+    results = await Promise.all(operations.map(operationHandler));
+  } catch (error) {
+    // Unhandled error, flag with Airbrake
+    return next({
+      error,
+      message: `Failed to sanitise application data. ${(error as Error).message}`,
+    });
+  }
+  await postToSlack(results);
```

Comment:
Is there value in posting the successes to Slack in this case, or do we really just care about visiblity of failed CRON jobs? I totally understand the little confidence boost that comes from an affirmative notification and am fine with both at first, but imagine it may start to feel noisy quick.

Reply:
You're bang on the money here - we really don't want Slack post from every PR, plus staging and prod each night...! I'll change this to only post failures, and just manually check it for the first couple of days after it's merged 👍